core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -305,6 +311,27 @@ public int numConnections() {
         return connectionLimitingHandler.numConnections();
     }
 
+    /**
+     * Setup version metrics
+     */
+    @VisibleForTesting
+    void setupVersionMetrics() {
+        final MeterRegistry meterRegistry = config().meterRegistry();
+        final Map<String, Version> map = Version.identify(getClass().getClassLoader());
+        final Version versionInfo = map.get("armeria");
+        final String version = versionInfo.artifactVersion();
+        final String commit = versionInfo.longCommitHash();
+        final String repositoryStatus = versionInfo.repositoryStatus();
+        final List<Tag> tags = ImmutableList.of(Tag.of("version", version),
+                                                Tag.of("commit", commit),
+                                                Tag.of("repostatus", repositoryStatus));
+        Gauge.builder("armeria.build.info", () -> 1)
+             .tags(tags)
+             .description("A metric with a constant '1' value labeled by version and commit hash"
+                          + " from which Armeria was built.")
```

Comment:
Armeria uses a slightly different convention for infix operators with line wrapping.
```java
             .description("A metric with a constant '1' value labeled by version and commit hash" +
                          " from which Armeria was built.")
```